arc_data_services/arcdata-deployer.yaml

```diff
@@ -197,6 +197,7 @@ rules:
   verbs:
   - delete
```

Comment:
Delete permissions aren't needed by the job anymore

Reply:
That will break existing release that's already out there.  Also, don't we need that for uninstall?

Reply:
Oh I thought this stuff was for new deployments, but not sure about the uninstall piece. The webhook job doesn't do any deleting, just patching. But yeah if these permissions cover more than the webhook job then we probably need it

Reply:
I see your PR in arcdata still has "delete" there.